dkist/dataset/tiled_dataset.py

```diff
@@ -181,10 +171,18 @@ def _validate_component_datasets(datasets, inventory):
         for ds in datasets[1:]:
             if ds.wcs.world_axis_physical_types != pt_1:
                 raise ValueError("The physical types do not match between all datasets")
-            if ds.meta["inventory"] and ds.meta["inventory"] is not inventory:
+            if ds.meta["inventory"] and ds.meta["inventory"] != inventory:
```

Comment:
I'd rather not make this change. The "is not" was deliberate, it should be the exact same dict in all places as it's the same record.

Reply:
Does it matter if the content is the same?

Reply:
it matters if you save it back to asdf as it wont dedupe. Also it's not as technically correct.